src/main/resources/static/js/service-center.js

```
@@ -0,0 +1,324 @@
+        // 정렬 버튼 이벤트
+        document.addEventListener('click', (e) => {
+            if (e.target.classList.contains('sort-btn')) {
+                this.handleSortClick(e.target);
+            }
+        });
+
+        // 카센터 카드 클릭 이벤트
+        document.addEventListener('click', (e) => {
+            const card = e.target.closest('.service-center-card');
+            if (card && card.dataset.centerId) {
+                this.navigateToDetail(card.dataset.centerId);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)